bigframes/ml/llm.py

```diff
@@ -79,6 +79,16 @@
     _GEMINI_1P5_PRO_FLASH_PREVIEW_ENDPOINT,
     _GEMINI_2_FLASH_EXP_ENDPOINT,
 )
+_GEMINI_FINE_TUNE_ENDPOINTS = (
+    _GEMINI_PRO_ENDPOINT,
+    _GEMINI_1P5_PRO_002_ENDPOINT,
+    _GEMINI_1P5_FLASH_002_ENDPOINT,
+)
+_GEMINI_SCORE_ENDPOINTS = (
```

Comment:
Can we merge the 2 lists? They are identical, don't think they will diverge.

Reply:
done